ovirt_imageio/client/_options.py

```diff
@@ -15,6 +15,7 @@
 
 from .. _internal.units import KiB, MiB, GiB, TiB
 from . _api import MAX_WORKERS, BUFFER_SIZE
+from . import __version__
```

Comment:
This can create import cycle (you depend on __init__.py, which may depend on other module which may depend on this module. If you are lucky this works, but when it breaks it is ugly. The only sane way is to always avoid these kinds of imports.

This also use public API when to implement private API. This is always bad idea and 
should be avoided. Never call public API from private API of from another public API. Public API should call only private APIs.

Using:

    from .. _internal import version

is the canonical way to get the version. Please use it like other code accessing the version.

Reply:
ACK